lib/kadai-core/src/main/java/io/kadai/task/api/TaskService.java

```diff
@@ -893,6 +893,246 @@ BulkOperationResults<String, KadaiException> transferTasksWithOwner(
           WorkbasketNotFoundException,
           NotAuthorizedOnWorkbasketException;
 
+  /**
+   * Distributes {@linkplain Task} instances from a source {@linkplain Workbasket} to one or more
+   * destination {@linkplain Workbasket}s based on a custom distribution strategy specified by its
+   * name.
+   *
+   * @param sourceWorkbasketId The {@linkplain Workbasket#getId() Id} of the source workbasket
+   *     containing the tasks to be distributed.
+   * @param taskIds A list of task IDs to be distributed. All tasks must belong to the source
+   *     workbasket.
+   * @param destinationWorkbasketIds A list of {@linkplain Workbasket#getId() Ids} of the
+   *     destination workbaskets where tasks will be distributed.
+   * @param distributionStrategyName The name of the custom distribution strategy to be applied. The
+   *     strategy must be registered in the SPI. If {@code null}, the default strategy is applied.
+   * @param additionalInformation A map containing additional context-specific information for the
+   *     distribution strategy. This parameter may be {@code null}.
+   * @return BulkResult with {@linkplain Task#getId() ids} and Error for each failed transaction.
```

Comment:
maybe operation is the better term as transaction? Basically we are transferring more than one task within one transaction, right?